website/pages/docs/cloud-init/getting-started.mdx

```diff
@@ -0,0 +1,145 @@
+---
+layout: docs
+page_title: Vagrant Cloud-Init Getting Started
+sidebar_title: Getting Started
+description: Walkthrough of a basic cloud-init configuration in Vagrant
+---
+
+## Prerequisites
+
+### Vagrant box with cloud-init installed
+
+Since cloud-init configures VMs the first time they are started, it will only
+work with Vagrant boxes that come with cloud-init preinstalled.
+Unfortunately, it's not sufficient to install cloud-init using a provisioner,
+because provisioners don't run until after SSH communication can be
+established -- much later in the boot process.  Therefore we recommend
+creating a base box with [Packer](https://www.packer.io/) as the starting
+point for working with cloud-init.
```

Comment:
We could maybe also recommend the `vagrant package` command with a cloud-init provisioner?